src/__tests__/unit/middleware/api-middleware.test.ts

```
@@ -44,11 +44,14 @@ jest.mock('@/lib/billing/manager', () => ({
   billingManager: mockBillingManager
 }));
 
+import { TestUser, TestOrganization, createTestUser, createTestOrganization, createTestSubscriptionPlan, createTestSubscription } from '@/__tests__/utils/test-helpers';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'createTestUser'. Consider to delete it or rename it.

'createTestUser' is defined here:


(lint/suspicious/noRedeclare)